src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -279,6 +286,9 @@ def create_quantized_param(
 
             quantize_(module, self.quantization_config.get_apply_tensor_subclass())
 
+    def transform_state_dict(self, tensor_data, metadata):
```

Comment:
nit: I think we can make this a bit more clear, e.g. `transform_state_dict_before_saving`